hypha/apply/projects/views/report.py

```diff
@@ -1,7 +1,6 @@
 from django.contrib.auth.decorators import login_required
 from django.contrib.auth.mixins import UserPassesTestMixin
 from django.core.exceptions import PermissionDenied
-from django.http import Http404
```

Comment:
Replaces 404 with PermissionDenied since that is what seems commonly used in Hypha. Any special reason a 404 was used here?

Reply:
PermissionDenied seems better than 404 here as these pages are already behind the login.